### DISCUSSION TITLE
a few questions

### DISCUSSION THREAD

**Discussion by umut2054** [orioncactus/pretendard#138] [General]
Hello, I like your font, I have a few questions for you. Do you think Pretendard is the best version of Pretendard right now? Will there be updates to the font? Are you planning to release a "Display" version? Thank you

**Comment by orioncactus** [comment]
Hello! I've also gotten this question via email, so I'm answering this here as well.

Pretendard is currently in a relatively stable state, aside from some font build quality issues. The next update will address improvements in punctuation and numeric shapes, but overall, there isn't a significant need for further enhancements at this time.

Regarding your mention of the 'Screen' version, are you referring to the display version? If so, the answer is no. Pretendard occupies a middle ground between a display and text font, same to the base font, Inter. When comparing its overall shapes to SF Pro, it can be considered at a level around 20pt in optical size. Therefore, I believe it is currently capable of meeting the requirements for both display and text scenarios. Before developing the display family, my current focus is on CJK (Chinese, Japanese, Korean) language support.

If you need the display family of Pretendard, it would be helpful if you could specify which aspects you require it for. Your feedback will greatly assist me in planning the roadmap.
